mmv1/templates/terraform/examples/firebase_app_hosting_backend_basic.tf.tmpl

```diff
@@ -0,0 +1,76 @@
+# Service account setup only required once per project.
```

Comment:
For this example, are you intending for users to use this as a reference and potentially copy-and-paste this config as a canonical example to work from? I just couldn't tell, but if it is not suitable for documentation, you may want to rework things so that it does serve that audience (and you may want test-only or docs-only examples).

Reply:
Yes, there are some setup that's required only once per project, as opposed to once per backend. It's common for users to just copy-pasta code, so I want them to at least copy everything they need.